src/components/AppLayout/Header/components/Notifications/index.tsx

```diff
@@ -143,18 +152,26 @@ const NotificationsPopper = styled(Paper)`
   border-radius: ${sm};
   box-shadow: 0 0 10px 0 rgba(33, 48, 77, 0.1);
   width: 438px;
-  padding: 30px 23px;
+  padding: 24px 23px;
 `
 
 const NotificationsHeader = styled.div`
-  height: 30px;
   display: flex;
   align-items: flex-end;
   justify-content: space-between;
   margin-bottom: 16px;
 `
 
+const StyledDivier = styled(Divider)`
+  width: 438px;
+  height: 2px;
+  background-color: ${background};
+  width: 406px;
+  margin: 16px -23px;
```

Comment:
Same as above and width is also set twice.

Reply:
I've noticed that this isn't the correct width. I'll adjust it.

Reply:
After our discussion: I will come back to the 'semantic' structure of this and convert it to being `100%` width in the clarification epic.